src/core/weights/nodeWeights/nodeWeightsT.test.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+import {type NodeWeightsT, empty, copy, merge} from "./nodeWeightsT";
+import {NodeAddress} from "../../graph";
+
+export function simpleNodeWeightsT(
+  nodeWeightsT: [string, number][]
+): NodeWeightsT {
+  const nw = empty();
+  for (const [addrPart, weight] of nodeWeightsT) {
+    nw.set(NodeAddress.fromParts([addrPart]), weight);
+  }
+  return nw;
+}
+
+describe("core/weights/edgeWeights/edgeWeightsT", () => {
```

Comment:
I think this might be at least part of the issue with your tests

```suggestion
describe("core/weights/nodeWeights/nodeWeightsT", () => {
```